### PR TITLE
[MIG] mail_activity_board: Migration to 14.0

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

**Comment by atchuthan** [comment]
/home/runner/work/social/social/mail_activity_board/static/src/components/chatter_topbar/chatter_topbar.js
  28:33  error  'ev' is defined but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)

>> Please let me know how to resolve this pre-commit as Odoo uses ev as parameter in the method
https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/components/chatter_topbar/chatter_topbar.js#L47-L63

File: mail_activity_board/views/mail_activity_view.xml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+                                                <span t-attf-class="text-danger">
+                                                    <field
+                                                        name="calendar_event_id_start"
+                                                        t-options='{"widget": "date"}'
+                                                    />
+                                                </span>
+                                            </t>
+                                        </t>
+                                        <t t-else="">
+                                            <span><i class="fa fa-clock-o" /></span>
```
Comment (max3903) [inline mail_activity_board/views/mail_activity_view.xml]:
```
2021-03-07 21:30:56,685 218 WARNING odoodb odoo.addons.base.models.ir_ui_view: A <i> with fa class (fa fa-clock-o) must have title in its tag, parents, descendants or have text
```

Comment (atchuthan) [reply mail_activity_board/views/mail_activity_view.xml]: @max3903 the warning is resolved now

**Comment by emagdalenaC2i** [comment]
@SodexisTeam There is an issue with pre-commit. Please, could you fix it?

**Comment by atchuthan** [comment]
@emagdalenaC2i 
The definition is based on Odoo module. https://github.com/OCA/social/pull/661#issuecomment-765278859

Any suggestion is much appreciated.

**Review by manuelregidor — APPROVED** [review]
It seems to be working fine.

**Review by ypapouin — CHANGES_REQUESTED** [review]
LGTM - Code Review
Please fix pre-commit

**Comment by atchuthan** [comment]
pre-commit fixed and squashed commits.
Ready for review.
@emagdalenaC2i @max3903 @ypapouin @pedrobaeza

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: mail_activity_board/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+* `SDI <https://www.sdi.es>`_:
+
+  * David Juaneda
+
+* `Eficent <https://www.eficent.com>`_:
+
+  * Miquel Raïch (miquel.raich@eficent.com)
```
Comment (MiquelRForgeFlow) [inline mail_activity_board/readme/CONTRIBUTORS.rst]:
```suggestion
* `ForgeFlow <https://www.forgeflow.com>`_:

  * Miquel Raïch (miquel.raich@forgeflow.com)
```

File: mail_activity_board/models/mail_activity.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
+# Copyright 2018 David Juaneda - <djuaneda@sdi.es>
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
```
Comment (MiquelRForgeFlow) [inline mail_activity_board/models/mail_activity.py]:
```suggestion
# Copyright 2018 ForgeFlow S.L.  <https://www.forgeflow.com>
```

**Review by MiquelRForgeFlow — COMMENTED** [review]
Please :)

**Review by ypapouin — APPROVED** [review]
LGTM

**Review by HaraldPanten — APPROVED** [review]
LGTM if you change Eficent/Forgeflow 👍

**Comment by atchuthan** [comment]
> LGTM if you change Eficent/Forgeflow

Done changes

**Review by angelmoya — APPROVED** [review]
:+1:

**Review by emagdalenaC2i — APPROVED** [review]

**Review by gaikaz — APPROVED** [review]

**Review by atchuthan — APPROVED** [review]

**Pull request merged**